dolphinscheduler-task-plugin/dolphinscheduler-task-datasync/src/test/java/org/apache/dolphinscheduler/plugin/task/datasync/DatasyncTaskTest.java

```diff
@@ -0,0 +1,234 @@
+ */
+
+package org.apache.dolphinscheduler.plugin.task.datasync;
+
+import static org.mockito.Mockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.doReturn;
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.api.mockito.PowerMockito.spy;
+import static org.powermock.api.mockito.PowerMockito.when;
+
+import org.apache.dolphinscheduler.plugin.task.api.TaskExecutionContext;
+import org.apache.dolphinscheduler.spi.utils.JSONUtils;
+import org.apache.dolphinscheduler.spi.utils.PropertyUtils;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+
+import static org.mockito.Mockito.any;
+
+import org.powermock.api.support.membermodification.MemberModifier;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import software.amazon.awssdk.http.SdkHttpResponse;
+import software.amazon.awssdk.services.datasync.DataSyncClient;
+import software.amazon.awssdk.services.datasync.model.CancelTaskExecutionRequest;
+import software.amazon.awssdk.services.datasync.model.CancelTaskExecutionResponse;
+import software.amazon.awssdk.services.datasync.model.CreateTaskRequest;
+import software.amazon.awssdk.services.datasync.model.CreateTaskResponse;
+import software.amazon.awssdk.services.datasync.model.StartTaskExecutionRequest;
+import software.amazon.awssdk.services.datasync.model.StartTaskExecutionResponse;
+import software.amazon.awssdk.services.datasync.model.TaskExecutionStatus;
+import software.amazon.awssdk.services.datasync.model.TaskStatus;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+        JSONUtils.class,
+        PropertyUtils.class,
+        DatasyncHook.class
+})
+@PowerMockIgnore({"javax.*"})
+public class DatasyncTaskTest {
+
+    private static final String mockExeArn = "arn:aws:datasync:ap-northeast-3:523202806641:task/task-017642db08fdf6a55/execution/exec-0ac3607778dfc31f5";
+
+    private static final String mockTaskArn = "arn:aws:datasync:ap-northeast-3:523202806641:task/task-071ca64ff4c2f0d4a";
+
+    DatasyncHook datasyncHook;
+
+    DatasyncTask datasyncTask;
+
+    @Mock
+    DataSyncClient client;
+    MockedStatic<DatasyncHook> datasyncHookMockedStatic;
+    @Before
+    public void before() throws IllegalAccessException {
+        client = mock(DataSyncClient.class);
+        datasyncHookMockedStatic = mockStatic(DatasyncHook.class);
+        when(DatasyncHook.createClient()).thenReturn(client);
+
+        DatasyncParameters DatasyncParameters = new DatasyncParameters();
+        datasyncTask = initTask(DatasyncParameters);
+        MemberModifier.field(DatasyncTask.class, "hook").set(datasyncTask, datasyncHook);
+    }
+
+    @Test
+    public void testCreateTaskJson() {
```

Comment:
What is the purpose of this test case?  If just test the value from json, it seems meanless.

Reply:
the purpose is to confirm the transformation of  `using JSON data to construct data sync task`, and validate the way to convert JSON data to an entity.